contributing/index.md

```diff
@@ -20,15 +20,15 @@ Data Commons welcomes patches and contributions to this project from everyone, n
 
 ### Add data
 
-We welcome contributions of public data to the Data Commons Graph. We’ve bootstrapped the graph with [data](/datasets) from US Census, World Bank, CDC, NOAA, IPCC, etc. However, Data Commons is meant to be for the community, by the community. We're excited to work with you to make public data accessible to everyone. Please see [the Data Commons data repository](https://github.com/datacommonsorg/data) for more information, and a [list of imports](https://github.com/orgs/datacommonsorg/projects/10/views/1) that would be helpful to add.
+We welcome contributions of public data to the Data Commons Graph. Data added will be accessible via Data Commons tools and APIs. We’ve bootstrapped the graph with these [datasets](/datasets) from US Census, World Bank, CDC, NOAA, NASA, etc. However, Data Commons is meant to be for the community, by the community. We're excited to work with you to make public data accessible to everyone. To get started, please take a look at [these resources](https://github.com/datacommonsorg/data/tree/master/docs) and follow [this development process](https://github.com/datacommonsorg/data). Here is [list of imports](https://github.com/orgs/datacommonsorg/projects/10/views/1) that would be helpful to add.
```

Comment:
minor nit: include link to cover "these"?  [these datasets](/datasets)

Reply:
nice catch, done